### PR TITLE
move chainbase related functions from Manager to ChainbaseManager

### PR DESCRIPTION
**Pull request by tronalex**
**What does this PR do?**
1. move chainbase related functions from Manager to ChainbaseManager
2. remove redundant Synchronization performed on LinkedBlockingQueue.

**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

**Comment by tronalex** [comment]
Who can help me about the build failure. https://sonarcloud.io/project/issues?id=java-tron&issues=AW_FduP3SMikdCpWcWaW&open=AW_FduP3SMikdCpWcWaW&pullRequest=2899 
It's obviously a false alarm. Please check.

File: framework/src/main/java/org/tron/core/db/Manager.java [hunk 1]
```diff
@@ -1049,19 +1040,18 @@ public synchronized void pushBlock(final BlockCapsule block)
       logger.info(SAVE_BLOCK + newBlock);
     }
     //clear ownerAddressSet
-    synchronized (pushTransactionQueue) {
```
Comment (wubinTron) [inline framework/src/main/java/org/tron/core/db/Manager.java]: why delete synchronized (pushTransactionQueue)

Comment (tronalex) [reply framework/src/main/java/org/tron/core/db/Manager.java]: > why delete synchronized (pushTransactionQueue)
BlockingQueue implementations are thread-safe. All queuing methods are atomic in nature and use internal locks.

**Comment by tronalex** [comment]
@wubinTron Please help to solve the sonar issue. https://sonarcloud.io/dashboard?id=java-tron&pullRequest=2899.

**Pull request merged**